src/main/java/com/amazon/opendistroforelasticsearch/ad/model/DetectorValidationIssueType.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package com.amazon.opendistroforelasticsearch.ad.model;
+
+import java.util.Collection;
+import java.util.Set;
+
+import com.amazon.opendistroforelasticsearch.ad.Name;
+
+public enum DetectorValidationIssueType implements Name {
```

Comment:
Is this class only to indicate the wrong field?

Reply:
For now, yes. For model level validation, we may add new field like "memory_size"